templates/pom.xml

```diff
@@ -1,19 +1,19 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>com.devonfw.java.dev</groupId>
-    <artifactId>devon4j</artifactId>
-    <version>dev-SNAPSHOT</version>
-  </parent>
-  <artifactId>devon4j-templates</artifactId>
-  <packaging>pom</packaging>
-  <name>${project.artifactId}</name>
-  <description>Templates (maven archetypes) of devon4j.</description>
-
-  <modules>
-    <module>server</module>
-  </modules>
-
-</project>
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
same here? No change at all?

Reply:
I will review it again and restore the file if changes not required